scripts/Makefile.am

```diff
@@ -0,0 +1,24 @@
+## This file is part of Invenio.
+## Copyright (C) 2016 CERN.
+##
+## Invenio is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## Invenio is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with Invenio; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+EXTRA_DIST = \
+  create-instance.sh \
+	drop-instance.sh \
```

Comment:
Whoops, wrong untabify, it seems.  I'll fix this, and add a note about Vagrant to the INSTALL file, and add sudo rights for Apache to issue commands as nobody for OpenOffice, and re-push.